ts/components/ui/Markdown/index.tsx

```diff
@@ -150,10 +150,24 @@ const generateCustomFontList = () => {
   </style>`;
 };
 
+const avoidTextSelectCSS = () => {
```

Comment:
Could it be a string const instead of a function?

Reply:
I uses a function to make it similar to the `generateCustomFontList = () => {` that acts similar to the new feature on a different style props, maybe we should change it as well? Or at least create one common function to generate the full style tag?

Reply:
@CrisTofani 
I like the first proposal you done: change all other functions to **const strings** if they don't need to be function (if their output is always the same and they don't need any input, they should be constants)

In the other hand I prefer to keep a fine granularity in the css composition, in place of one common function. I guess it's more readable. What do you think?

Reply:
What i thought was that we could summarize in one single function this part of code in `generateHtml` func:

```javascript
  ${cssStyle ? generateInlineCss(cssStyle) : ""}
  ${avoidTextSelection ? avoidTextSelectionCSS : ""}
  ${useCustomSortedList ? generateCustomFontList : ""}
```
creating a function to generate our _optional CSS_

Anyway i don't think this edit would improve readability, so i guess we can use the **const string** without no problem!